asana/resources/gen/portfolios.py

```diff
@@ -0,0 +1,193 @@
+        their own starting state on a portfolio.
+
+        Parameters
+        ----------
+        [data] : {Object} Data for the request
+          - workspace : {Gid} The workspace or organization in which to create the portfolio.
+          - name : {String} The name of the newly-created portfolio
+          - [color] : {String} An optional color for the portfolio
+        """
+        return self.client.post("/portfolios", params, **options)
+        
+    def find_by_id(self, portfolio, params={}, **options): 
+        """Returns the complete record for a single portfolio.
+
+        Parameters
+        ----------
+        portfolio : {Gid} The portfolio to get.
+        [params] : {Object} Parameters for the request
+        """
+        path = "/portfolios/%s" % (portfolio)
+        return self.client.get(path, params, **options)
+        
+    def update(self, portfolio, params={}, **options): 
+        """An existing portfolio can be updated by making a PUT request on the
+        URL for that portfolio. Only the fields provided in the `data` block will be
+        updated; any unspecified fields will remain unchanged.
+        
+        Returns the complete updated portfolio record.
+
+        Parameters
+        ----------
+        portfolio : {Gid} The portfolio to update.
+        [data] : {Object} Data for the request
+        """
+        path = "/portfolios/%s" % (portfolio)
+        return self.client.put(path, params, **options)
+        
+    def delete(self, portfolio, params={}, **options): 
+        """An existing portfolio can be deleted by making a DELETE request
+        on the URL for that portfolio.
+        
+        Returns an empty data record.
+
+        Parameters
+        ----------
+        portfolio : {Gid} The portfolio to delete.
+        """
+        path = "/portfolios/%s" % (portfolio)
+        return self.client.delete(path, params, **options)
+        
+    def find_all(self, params={}, **options): 
+        """Returns a list of the portfolios in compact representation that are owned
+        by the current API user.
+
+        Parameters
+        ----------
+        [params] : {Object} Parameters for the request
+          - workspace : {Gid} The workspace or organization to filter portfolios on.
+          - owner : {String} The user who owns the portfolio. Currently, API users can only get a
+          list of portfolios that they themselves own.
+        """
+        return self.client.get_collection("/portfolios", params, **options)
+        
+    def get_items(self, portfolio, params={}, **options): 
+        """Get a list of the items in compact form in a portfolio.
+
+        Parameters
+        ----------
+        portfolio : {Gid} The portfolio from which to get the list of items.
+        [params] : {Object} Parameters for the request
+        """
+        path = "/portfolios/%s/items" % (portfolio)
+        return self.client.get(path, params, **options)
```

Comment:
I'm stunned I noticed this because it's so easy to miss, but this should be `get_collection` and not `get`. That's one of the easy-to-miss things in our generated code :/